app/Main.hs

```diff
@@ -41,8 +46,8 @@ main = do
     Web.Environment     {..} = env |> web
     AWS.Environment     {..} = env |> aws
 
-    herokuID       = Hku.ID       <| encodeUtf8 (manifest |> Hku.id)
-    herokuPassword = Hku.Password <| encodeUtf8 (manifest |> Hku.api |> Hku.password)
+    herokuID       = Hku.ID       <| encodeUtf8 <| Hku.id <| manifest
+    herokuPassword = Hku.Password <| encodeUtf8 <| Hku.password <| Hku.api <| manifest
```

Comment:
Not trying to pun on dot syntax. The theory is that pipe directionality is actually a stronger factor in reading here.